cli/src/main/java/org/jboss/galleon/cli/HelpSupport.java

```diff
@@ -43,100 +47,18 @@
-        staticContent = staticContent.replace(AVAILABLE_COMMANDS, availableCommands);
-        return staticContent;
-    }
-
-    private static String formatDomains(Collection<HelpCommandDomain> domains, Shell shell) {
-        StringBuilder commands = new StringBuilder();
-        for (HelpCommandDomain domain : domains) {
-            commands.append(Config.getLineSeparator()).append(domain.domain.getDescription()).
-                    append(Config.getLineSeparator());
-            commands.append(format(domain.commands, shell.size().getWidth()));
-        }
-        return commands.toString();
-    }
-
-    private static String format(Map<String, String> options, int width) {
-        int max = 0;
-        List<String> keys = new ArrayList<>();
-        for (String k : options.keySet()) {
-            if (k.length() > max) {
-                max = k.length();
-            }
-            keys.add(k);
-        }
-        Collections.sort(keys);
-        StringBuilder builder = new StringBuilder();
-        builder.append(Config.getLineSeparator());
-        int remaining = width - max - TAB.length();
-        String maxPad = pad(max + TAB.length(), "");
-        for (String k : keys) {
-            String val = options.get(k);
-            builder.append(pad(max, k)).append(TAB);
-            if (val.length() <= remaining) {
-                builder.append(val).append(Config.getLineSeparator());
-            } else {
-                String pad = "";
-                do {
-                    int limit = Math.min(remaining, val.length());
-                    String content = val.substring(0, limit);
-                    val = val.substring(limit);
-                    String toNextLine = null;
-                    if (!val.isEmpty()) {
-                        // cut at the last whitespace
-                        int i = content.lastIndexOf(" ");
-                        if (i >= 0) {
-                            toNextLine = content.substring(i += 1);
-                            content = content.substring(0, i);
-                            val = toNextLine + val;
-                        }
-                    }
-                    builder.append(pad).append(content).append(Config.getLineSeparator());
-                    pad = maxPad;
-                } while (val.length() > 0);
-            }
-        }
-        return builder.toString();
-    }
-
-    private static String pad(int max, String k) {
-        StringBuilder builder = new StringBuilder();
-        builder.append(k);
-        for (int i = k.length(); i < max; i++) {
-            builder.append(" ");
-        }
-        return builder.toString();
+            CommandRegistry<? extends Command, ? extends CommandInvocation> registry, Shell shell) throws IOException, CommandNotFoundException {
+        StringBuilder sb = new StringBuilder();
+        sb.append("== NOMINAL MODE ==").append(Config.getLineSeparator());
```

Comment:
NOMINAL perhaps DEFAULT.

Reply:
What about NORMAL? DEFAULT sounds wrong to be for some reason.